packages/blockchain-extension/test/explorer/environmentExplorer.test.ts

```diff
@@ -255,14 +253,14 @@ describe('environmentExplorer', () => {
                 await FabricEnvironmentRegistry.instance().add(localFabricEnv);
                 await FabricEnvironmentRegistry.instance().add(registryEntryOne);
                 await FabricEnvironmentRegistry.instance().add(registryEntryTwo);
-                await FabricEnvironmentRegistry.instance().add(managedAnsible);
+                // await FabricEnvironmentRegistry.instance().add(managedAnsible);
```

Comment:
A few lines were left commented out in this file, probably better removed. I won't add a comment to each though.